yarn.lock

```diff
@@ -6472,8 +6472,8 @@ class-utils@^0.3.5:
 
 classnames@^2.2.5, classnames@^2.2.6:
   version "2.2.6"
-  resolved "https://registry.npmjs.org/classnames/-/classnames-2.2.6.tgz#43935bffdd291f326dad0a205309b38d00f650ce"
-  integrity sha512-JR/iSQOSt+LQIWwrwEzJ9uk0xfN3mTVYMwt1Ir5mUcSN6pU+V4zQFFaJsclJbPuAUQH+yfWef6tm7l1quW3C8Q==
+  resolved "https://artifactory.spotify.net/artifactory/api/npm/virtual-npm/classnames/-/classnames-2.2.6.tgz#43935bffdd291f326dad0a205309b38d00f650ce"
```

Comment:
noooooope :p

Looks like you could just revert this change though, no new versions.

Still searching for a proper solution to this >_>

Reply:
There seem to exist some lock file linters. Let's see if one supports limiting the resolved field soon

Candidate: https://github.com/lirantal/lockfile-lint/blob/master/README.md

Reply:
Yeah it's a bit surprising that the build goes through